### DISCUSSION TITLE
part-ii/2021-47/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#370] [Question comments]
Part II 2021 Differential Geometry: Paper 1, Section II, 26F 

https://questions.tripos.org/part-ii/2021-47/

**Comment by ArthurConmy** [comment]
<details>
<summary>cii)</summary>

0 everywhere, since $\phi(y, \theta) = ( y(1+\cos \theta) ,  y , y( 1 - \sin \theta ) )^T $ and so normal is $y$ independent so $K=0$.

</details>
